Docs/source/install/hpc/juwels.rst

```diff
@@ -69,10 +65,26 @@ Then, ``cd`` into the directory ``$HOME/src/warpx`` and use the following comman
 
 .. code-block:: bash
 
-   make -j 16 COMP=gcc USE_GPU=TRUE MPI_THREAD_MULTIPLE=FALSE
+   cd $HOME/src/warpx
+   rm -rf build
+
+   cmake -S . -B build -DWarpX_COMPUTE=CUDA -DWarpX_OPENPMD=ON -DWarpX_MPI_THREAD_MULTIPLE=OFF
+   cmake --build build -j 16
 
 The other :ref:`general compile-time options <install-developers>` apply as usual.
 
+The executable will be generated in ``build/bin/``.
+
+.. note::
+
+   Currently, if you want to use HDF5 output with openPMD, you need to add
+
+   .. code-block:: bash
+
+      export OMPI_MCA_io=romio321
+
+   in your job scripts, before running the ``srun`` command.
+
 Running
 -------
 
```

Comment:
@MaxThevenet does Juwels still have the V100 GPUs *and* also has the A100 GPUs now?

We can probably update this further in a follow-up PR :)

Reply:
Yes Juwels still has V100 nodes, and the A100 nodes are in the Juwels Booster. Sure, we will add the Juwels Booster in there at some point.